h2o-docs/src/product/getting-data-into-h2o.rst

```diff
@@ -256,6 +256,7 @@ The ``import_sql_table`` function accepts the following parameters:
 - ``password``: The password for SQL server
 - ``optimize``: Specifies to optimize the import of SQL table for faster imports. Note that this option is experimental.
 - ``fetch_mode``: Set to DISTRIBUTED to enable distributed import. Set to SINGLE to force a sequential read by a single node from the database.
+- ``num_chunks_hint``: Desired number of chunks for the target Frame.
```

Comment:
Can you change this to: "Optionally specify the number of chunks for the target frame."